src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -96,7 +96,6 @@
 
 .schedule-list__action-list-cell {
   padding: 4px 8px;
-  width: calc(68% - 84px);
```

Comment:
After making the other changes, turns out I didn't need to set a width.

And, also, I was mistakenly applying this width to each cell in the table. Not just the `State, Action` cell I was intending to apply it to 🤦

So I removed it.